README.md

```diff
@@ -5,21 +5,76 @@
 cri-containerd is a [containerd](https://containerd.io/) based implementation of Kubernetes [container runtime interface (CRI)](https://github.com/kubernetes/kubernetes/blob/v1.6.0/pkg/kubelet/api/v1alpha1/runtime/api.proto).
 ![cri-containerd](./docs/cri-containerd.png)
 ## Current Status
-We are currently working toward an alpha version to be used in conjunction with Kubernetes 1.7. See the [roadmap](./docs/proposal.md#roadmap-and-milestones) for information about current and future milestones.
+The current release of `cri-containerd` is `v0.1.0-alpha.1`. This release is for
+use with Kubernetes 1.7. See the [roadmap](./docs/proposal.md#roadmap-and-milestones)
+for information about current and future milestones.
+## Social
+This is a project within the Kubernetes SIG named `sig-node`.
+For more information about `sig-node` and the `cri-containerd` project:
+* [sig-node community site](https://github.com/kubernetes/community/tree/master/sig-node).
+* Catch us on Kubernetes (kubernetes.slack.com) in the `#sig-node` channel.
+* Mailing List: https://groups.google.com/forum/#!forum/kubernetes-sig-node
+## Getting started
+### Binary Dependencies
+The alpha release of cri-containerd has been tested on [containerd](https://github.com/containerd/containerd) v0.2.3-1098-g8ed1e24, [runc](https://github.com/opencontainers/runc) v1.0.0-rc3, and current releases of [CNI](https://github.com/containernetworking/cni).
+### Build cri-containerd
+1. Install and setup a go1.8.x development environment.
+1. Make a local clone of this repository.
+1. From your github.com/kubernetes-incubator/cri-containerd/ path (Note: run these commands as root):
+```
+make install.tools
+make
+make install
+```
+### Running with cri-tools
```

Comment:
How about we change the text to reflect your new make install.deps and make test-cri.. then just point them to cri-tools cli that they installed with make install.deps (presuming you agree that install.deps should have the cli-tools installed)..